src/PrestaShopBundle/Resources/config/services/core/grid/grid_data_factory.yml

```diff
@@ -10,6 +10,11 @@ services:
             - '@prestashop.core.grid.query.doctrine_query_parser'
             - 'logs'
 
+    prestashop.core.grid.factory.log_decorator:
+      class: PrestaShop\PrestaShop\Core\Grid\Data\Factory\LogsGridDataFactory
```

Comment:
The indentation is four spaces in this file.

Reply:
Fixed